test/arrays/userAPI/blockOps2D.comm-none.good

```diff
@@ -0,0 +1,414 @@
+Block-distributed array
+----------------
+eltType is: real(64)
+idxType is: int(64)
+rank is: 2
+
+size is: 100
+numElements is: 100
+shape is: (10, 10)
+
+X is:
+1.0 2.0 3.0 4.0 5.0 6.0 7.0 8.0 9.0 10.0
+11.0 12.0 13.0 14.0 15.0 16.0 0.0 0.0 0.0 0.0
```

Comment:
I don't think I understand your "impact every test in this directory" comment then...